### PR TITLE
Added Anti-Aliasing appearance option

### PR DESCRIPTION
**Pull request by peterAdams1014**
[AB#547](https://dev.azure.com/euclideon/57c3bd2a-8f94-4578-b195-2e4fa9d8f295/_workitems/edit/547)

### CODE REVIEW

File: src/gl/directx11/vcRenderShaders.cpp [hunk 1]
```diff
@@ -1405,3 +1405,326 @@ const char *const g_udGPURenderGeomGeometryShader = R"shader(
...
+  PS_OUTPUT main(PS_INPUT input)
+  {
+    PS_OUTPUT output;
+    float depth = texture1.Sample(sampler1, input.uv).x;
+
+    // only run FXAA on edges (simple edge detection)
+    float depth0 = texture1.Sample(sampler1, input.edgeSampleUV0).x;
+    float depth1 = texture1.Sample(sampler1, input.edgeSampleUV1).x;
+    float depth2 = texture1.Sample(sampler1, input.edgeSampleUV2).x;
+    float depth3 = texture1.Sample(sampler1, input.edgeSampleUV3).x;
```
Comment (DamianMadden) [inline src/gl/directx11/vcRenderShaders.cpp]: Are the parameters for this sampler set-up to clamp?

File: src/gl/opengl/vcRenderShaders.cpp [hunk 2]
```diff
@@ -1179,3 +1179,300 @@ const char *const g_udGPURenderGeomGeometryShader = FRAG_HEADER R"shader(
...
+
+void main()
+{
+	float depth = texture(u_depth, v_uv).x;
+
+  // only run FXAA on edges (simple edge detection)
+  float depth0 = texture(u_depth, v_edgeSampleUV0).x;
+  float depth1 = texture(u_depth, v_edgeSampleUV1).x;
+  float depth2 = texture(u_depth, v_edgeSampleUV2).x;
+  float depth3 = texture(u_depth, v_edgeSampleUV3).x;
```
Comment (DamianMadden) [inline src/gl/opengl/vcRenderShaders.cpp]: same as above, this sampler going to be configured to clamp uvs?

Comment (DamianMadden) [reply src/gl/opengl/vcRenderShaders.cpp]: I think it's repeat?

File: src/gl/directx11/vcRenderShaders.cpp [hunk 3]
```diff
@@ -1405,3 +1405,326 @@ const char *const g_udGPURenderGeomGeometryShader = R"shader(
     //OutputStream.RestartStrip();
   }
 )shader";
+
+const char *const g_FXAAVertexShader = R"shader(
```
Comment (OsiJr) [inline src/gl/directx11/vcRenderShaders.cpp]: Can you put in a comment above the literal with the license information, please?

File: src/gl/opengl/vcRenderShaders.cpp [hunk 4]
```diff
@@ -1179,3 +1179,300 @@ const char *const g_udGPURenderGeomGeometryShader = FRAG_HEADER R"shader(
     EndPrimitive();
   }
 )shader";
+
+const char *const g_FXAAVertexShader = FRAG_HEADER R"shader(
```
Comment (OsiJr) [inline src/gl/opengl/vcRenderShaders.cpp]: License here as well

File: src/gl/directx11/vcRenderShaders.cpp (hunk 1 above)
Comment (peterAdams1014) [reply src/gl/directx11/vcRenderShaders.cpp]: Yep!

File: src/gl/opengl/vcRenderShaders.cpp (hunk 2 above)
Comment (peterAdams1014) [reply src/gl/opengl/vcRenderShaders.cpp]: It is clamped.

**Pull request merged**